app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -166,10 +166,19 @@ export class CloneGithubRepository extends React.Component<
         groups={this.state.repositories}
         filterText={this.state.filterText}
         onFilterTextChanged={this.onFilterTextChanged}
+        renderNoItems={this.noMatchingRepositories}
       />
     )
   }
 
+  private noMatchingRepositories = function() {
+    return (
+      <div className="no-results-found">
+        Sorry, there doesn't seem to be any matching repositories
```

Comment:
The other areas of our application have the following message when an empty set is returned of the following form: _Sorry, I can't find that ..._. Could we stick to the format to ensure our apps messaging remains consistent? 

I think _Sorry I can't find that repository_ follows that structure.